lib/Mojolicious/Guides.pod

```diff
@@ -54,6 +54,21 @@ L<features|feature> are enabled, even if examples don't specifically mention it.
 Some modules, like L<Mojo::Base> and L<Mojolicious::Lite>, will enable them for
 you automatically, whenever they are used.
 
+=item Variable names
+
+Where possible, the documentation will try to use variable names in example code
+that describes whatever type of data should be passed through to or is returned
+from methods and functions. For instance C<$bytes> or C<$chars> to distinguish
+whether it is encoded bytes or decoded characters, C<$bool> to indicate whether
+a value only indicates true or false, the use of C<$c> to denote a
```

Comment:
I think that instead of "true" and "false", I think you should do `L<true|Mojo::JSON/true>` or something like that to indicate what kind of "true" and "false" we are talking about.

Reply:
That's not the kind of true and false we're talking about though. It's just Perl boolean context true and false.

Reply:
It was changed from saying "truthiness" to "true or false"; like Grinnz says it's just Perl boolean context, not JSON::PP::Boolean